### PR TITLE
README.md update docker instructions

### PR DESCRIPTION
**Pull request by techtonik**
See https://github.com/lxc/lxd/issues/3299

Signed-off-by: anatoly techtonik <techtonik@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -283,17 +283,6 @@ Yes. The easiest way to do that is using a privileged container:
...
-
-Note that the docker profile does not provide a network interface, so the
-common case will want to compose the default and docker profiles.
-
-Also note that Docker coming from [upstream](https://apt.dockerproject.org/repo) doesn't currently run as is inside the lxd container. Look at issue [#2621](https://github.com/lxc/lxd/issues/2621) for more details. You need to download the docker coming from Ubuntu (docker.io package) to get this working. So once you are in the lxd container run
-
-    sudo apt-get install -y docker.io runc containerd
-
-The container must be using the Ubuntu 1.10.2-0ubuntu4 or newer docker package.
+    lxc config set <container> security.nesting true
```
Comment (brauner) [inline README.md]: I'd rather have:

"In order to run Docker inside a LXD container the `security.nesting` property for the container should be set to `true`. No other changes should be necessary."

Comment (brauner) [inline README.md]: If you want you can leave in the example `lxc config set <container> security.nesting true` line in.

**Comment by stgraber** [comment]
@brauner +1 on updated wording

**Comment by techtonik** [comment]
Should be okay now.

**Comment by MarkusTeufelberger** [comment]
Can someone verify if this works on Ubuntu xenial-backports with HWE kernels?

**Comment by techtonik** [comment]
Not me. My host is 17.04.

**Comment by MarkusTeufelberger** [comment]
I had some AppArmor issues on 16.04 without the docker profile, that's why I'm asking.

**Comment by brauner** [comment]
@MarkusTeufelberger, that's a little vague. :) Can you please specify what exactly the issue is. :)

**Comment by MarkusTeufelberger** [comment]
I'll open a separate issue I guess... Editing the link in here later.

**Comment by MarkusTeufelberger** [comment]
Urgh, and of course it works without major issues in Vagrant! -.-'

**Pull request merged**